scripts/docsascode/buildDoc.sh

```diff
@@ -76,10 +76,16 @@ function build_doc {
           # merge new config on previous
           mergeYml $current_output_template_path/config.yml $current_output_template_path/yaml_tmp/config.yml
           mergeYml $current_output_template_path/dac_custom-theme.yml $current_output_template_path/yaml_tmp/dac_custom-theme.yml
-          
+
           # remove temp yaml folder
           rm -rf $current_output_template_path/yaml_tmp/
 
+          # Check if any fonts dir exists in the template and add any ttf file from it
+          if [ -d $output_path/fonts/ ]
+          then
+              cp -f $output_path/fonts/*.ttf  $(gem environment gemdir)/gems/asciidoctor-pdf-$ASCIIDOCTOR_PDF_VERSION/data/fonts/
```

Comment:
This copy should be performed once, after the end of the loop.

Reply:
Existing fonts in destination folder must be preserved (not overwritten).

Reply:
> This copy should be performed once, after the end of the loop.

Hum in that case it will miss the fonts that would be associated to one output (or the inheritance)
I'd be in favor to keep that in the loop.

Reply:
> Existing fonts in destination folder must be preserved (not overwritten).

OK

Reply:
Ok to keep it in the loop